src/core/components/courses/ProfileFollowUp.vue

```diff
@@ -75,38 +93,29 @@ export default {
               .map(ah => ({ user: ah.user, date: ah.date })))))
```

Comment:
Tu peux laisser le premier map sur la meme ligne que this.learners

Reply:
done